JenkinsFile_build_doc.groovy

```diff
@@ -84,21 +96,46 @@ timeout(time: 6, unit: 'HOURS') {
                             sh "tar -zxf ${ARCHIVE}"
                         }
 
-                        dir('eclipse_repo') {
-                            git branch: BRANCH, url: ECLIPSE_REPO
-                            sh 'rm -rf *'
-                            sh "cp -r ${WORKSPACE}/built_doc/site/* ."
-                            sh 'git status'
-                            STATUS = sh (script: 'git status --porcelain', returnStdout: true).trim()
-                            if ("x${STATUS}" != "x") {
-                                sh 'git add -A'
-                                sh "git commit -m 'Generated from commit: ${MERGE_COMMIT}'"
-                                sh "git push origin ${BRANCH}"
-                            }
-                        }
+						if (params.ghprbPullId) {
+							dir('openj9-staging') {
+								git branch: BRANCH, url: OPENJ9_STAGING
+								// SUE: Was `sh 'rm -rf *'`
+								// SUE: Check if the PR is already staged. If so, remove content. Else create a new directory. 
```

Comment:
I'm guessing this comment is temporary, but there's an extra whitespace at the end of the line.